types/vote_test.go

```diff
@@ -294,3 +294,53 @@ func TestVoteProtobuf(t *testing.T) {
 		}
 	}
 }
+
+var sink interface{}
+
+var protoVote *tmproto.Vote
+var sampleCommit *Commit
+
+func init() {
+	protoVote = examplePrecommit().ToProto()
+
+	lastID := makeBlockIDRandom()
+	voteSet, _, vals := randVoteSet(2, 1, tmproto.PrecommitType, 10, 1)
+	commit, err := makeCommit(lastID, 2, 1, voteSet, vals, time.Now())
+	if err != nil {
+		panic(err)
+	}
+	sampleCommit = commit
+}
```

Comment:
Is there a way structure these tests to avoid the init function/panics, and global variables?

Reply:
It is much better to initiate them once, otherwise if we run -count=N for benchmarks we end up unnecessarily initializing variables.